Data/models/CRUDProtocols.swift

```diff
@@ -35,7 +35,7 @@ extension Deletable where Self: NSManagedObject {
     }
     
     static func deleteAll(predicate: NSPredicate? = nil,
-                          context: WriteContext = .new,
+                          context: WriteContext = .new(false),
```

Comment:
We should use a named parameter like `.new(inMemory: false)`, `false` alone doesn't say much